Tests/BugsnagOnCrashTest.m

```diff
@@ -0,0 +1,99 @@
+//
+//  BugsnagOnCrashTest.m
+//  Tests
+//
+//  Created by Jamie Lynch on 23/04/2020.
+//  Copyright © 2020 Bugsnag. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+
+#import <Bugsnag/Bugsnag.h>
+
+@interface BugsnagEvent ()
+- (instancetype)initWithKSReport:(NSDictionary *)report;
+@end
+
+@interface BugsnagOnCrashTest : XCTestCase
+
+@end
+
+@implementation BugsnagOnCrashTest
+
+- (void)testEmptyData {
+    BugsnagEvent *event = [[BugsnagEvent alloc] initWithKSReport:@{}];
+    XCTAssertNil([event getMetadataFromSection:@"onCrash"]);
```

Comment:
Use constant for key

Reply:
Agreed to keep using string literal as this a public field, so we want to verify the constant name itself is correct and doesn't change over time